src/admin/controller.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @version    CVS: 1.0.0
+ * @package    Com_Admintask
+ * @author     Tasolglobal <testing@tasolglobal.com>
```

Comment:
Please replace the `Tasolglobal <testing@tasolglobal.com>` with `__COMPONENT_CONTACT__` in all files.